code/script.js

```diff
@@ -222,104 +230,205 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+   console.log(secret); //To check that a new secret person has been selected 
 }
 
-// This function to start (and restart) the game
-const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+// When the game starts:
+const start = () => { // Use this function to start (and restart) the game
+  charactersInPlay = CHARACTERS   // Here we're setting charactersInPlay array to be all the characters 
+  winOrLose.style.display = 'none' // We dont want to show the win/lose screen here 
+  board.style.display = 'flex' // We want to show the game board 
+  setSecret() // This sets a new secret person  
+  generateBoard() // This draws the board with all the people 
```

Comment:
I think it's great that you have left many comments throughout your code.  It's a great way to understand what you've done and why, but also will be super helpful when you revisit your code.

Reply:
Thanks Bridget!